lerna.json

```diff
@@ -3,6 +3,6 @@
     "packages/*"
   ],
   "useWorkspaces": true,
-  "version": "2.148.0-next.15",
+  "version": "2.148.0",
```

Comment:
@kevinsperrine @bjornalm is this version change correct?

Reply:
Comparing with this older commit for 2.147.0 it looks correct
https://github.com/carbon-design-system/carbon-addons-iot-react/commit/0bcc680eebf4398f001aa99df0625e5f28848dde